### DISCUSSION TITLE
Question about top bar of notifications

### DISCUSSION THREAD

**Discussion by ghost** [Windos/BurntToast#202] [Q&A]
Looking at some of the screenshots in the issues/discussions, I notice some of the notifications don't have the top bar containing the icon/app name. Is there a way to do this?

Also quick question: I've read a lot of the issues/discussions about registering powershell to open with a URI. But is there a way to launch a batch file, or execute a powershell command/send to powershell when a button is pressed without the URI method?

Thanks a lot

**Comment by Windos** [comment]
Re that "top bar" what you're likely seeing is due to some changes in Windows 10 over the years.

For example the default toast (when you just run `New-BurntToastNotification` used to look like this:

![Old default toast](https://user-images.githubusercontent.com/6955786/172088822-fe1da6f2-5337-427e-9820-c967ddfc84f2.png)

And now without any code changes, the only thing being a feature release or two for Windows 10, it now looks like this:

![New default toast](https://user-images.githubusercontent.com/6955786/172088880-53a631ce-22f8-48ec-b96b-e93f7b8348ad.png)

What used to hide that top bar was the "app logo override" which on these is that photo of some burnt toast (you'll note that the name of the app is down the bottom of the toast, in the old one I'd managed to make it show as "BurntToast" but you'd normally see it as "Windows PowerShell".)

At some point Microsoft decided they needed better visibility for users of what is sending notifications and so they made it so that the little icon and app name **always** shows and I'm not sure of a way to get rid of it.

-----

Re actions, it is possible to trigger actions inside of PowerShell itself on a button press but the limitation is that it requires PowerShell 7.1 and above.

I've got some examples here:

* https://github.com/Windos/Talks/blob/main/2021/02%20-%20February/23%20-%20DuPSUG%20-%20Croque%20Monsieur/Demos/08_Events.ps1
* https://github.com/Windos/Talks/blob/main/2021/02%20-%20February/23%20-%20DuPSUG%20-%20Croque%20Monsieur/Demos/09_SelectionBox.ps1
